configs/config.annotated.yaml

```diff
@@ -0,0 +1,226 @@
+#####################
+### Iris-api settings
+#####################
+server:
+  host: 0.0.0.0
+  port: 16649
+  disable_auth: True # allow logins by existing users, but without checking credentials
+
+db: &db
+  conn:
+    kwargs:
+      scheme: mysql+pymysql
+      user: root
+      password: ""
+      host: 127.0.0.1
+      database: iris
+      charset: utf8
+    str: "%(scheme)s://%(user)s:%(password)s@%(host)s/%(database)s?charset=%(charset)s"
+  query_limit: 500
+  kwargs:
+    pool_recycle: 3600
+    echo: False
+    pool_size: 100
+    max_overflow: 100
+    pool_timeout: 60
+
+## Change these to random long values when you run this in production
+user_session:
+  encrypt_key: 'abc'
+  sign_key: '123'
+
+supported_timezones:
+  - 'US/Pacific'
+  - 'US/Eastern'
+  - 'US/Central'
+  - 'US/Mountain'
+  - 'US/Alaska'
+  - 'US/Hawaii'
+  - 'Asia/Kolkata'
+  - 'Asia/Shanghai'
+  - 'UTC'
+
+healthcheck_path: /tmp/status
+
+## Custom webhooks can be created and put in src/iris/webhooks.
+## Refer to their class name in this array.
+## The webhook will be available at /v0/webhooks/$name and requires being called with
+## ?application=xxx&key=xxx
+#webhooks: []
+
+#####################
+### Iris-sender
+#####################
+sender:
+  debug: True # Set to True, sender will not actually send messages
+  process_title: iris-sender
+  host: 127.0.0.1
+  port: 2321
+
+  ## Number of gevent workers per mode we send
+  workers_per_mode:
+    email: 500
+    #hipchat: 10
+
+  ## Default sender to reach out to if we can't determine one programmatically
+  master_sender:
+    host: 127.0.0.1
+    port: 2321
+
+  api_host: http://localhost:16649
+
+  ## App used for sending notification messages + incidents
+  sender_app: iris
+  ## RPC access log
+  access_log:
+    maxBytes: 10485760
+    filename: './logs/sender_rpc.access.log'
+
+  ## Optionally, use zookeeper for sender high availability.
+  #zookeeper_cluster: localhost:2181
+
+  ## If you're not using zookeeper, the following determines
+  ## whether this sender is a master and you hard code in
+  ## slaves to be cycled through
+  is_master: True
+
+#slaves:
+#  - host: 127.0.0.1
+#    port: 2322
+#  - host: 127.0.0.1
+#    port: 2323
+
+sender_workers: 1000
+
+## Enabled modules for resolving complex user roles to individual usernames, processed
+## in the order listed here.
+role_lookups:
+  - user # support for 'role' being username
+  - mailing_list # mailing list config; see bottom of file
+  #- oncall # oncall service integration
+  - dummy # dummy role lookup used for testing purposes. Disable this in production.
+
+#oncall-api: https://oncall.dummy.com  # oncall url
+
+applications:
+ - dummy_app
+
+## Vendors provides a way to add support for new notification mechanisms
+vendors: []
+#- type: iris_smtp
+#  name: email
+#  smtp_server: 'smtp1.example.com'
+#  smtp_gateway: 'smtp2.example.com'
+#  from: 'foouser@example.com'
```

Comment:
Can you add, double commented out, `autoscale: False` and `tasks_per_mx: 4`, with a comment describing how, if enabled, these will make a pool of persistent connections per each MX record for the given smtp_gateway?

Reply:
Not entirely sure if I understand the double commented out correctly, but I've followed your instructions as literally as I could.